packages/ocs/dashboards/persistent-internal/details-card.tsx

```diff
@@ -117,12 +132,17 @@ const DetailsCard: React.FC = () => {
             {serviceVersion}
           </DetailItem>
           <DetailItem
-            key="inTransitEncryption"
-            title={t('In-transit encryption')}
+            key="encryption"
+            title={t('Encryption')}
             isLoading={!ocsLoaded}
-            error={ocsError as any}
+            error={ocsError}
           >
-            {inTransitEncryptionStatus}
+            <EncryptionPopover
+              encryptionDetails={encryptionDetails}
+              inTransitEncryption={!!inTransitEncryption}
```

Comment:
is `inTransitEncryption` boolean or string ??
```suggestion
              inTransitEncryption={inTransitEncryption}
```